scripts/ldap/base_configure_ldap.py

```diff
@@ -0,0 +1,209 @@
+#!/usr/bin/env python3
```

Comment:
This file needs to be merged in cortx-utils repo. Lets raise a separate PR on cortx-utils repo, Sachin Punadikar will be merging it to utils. We can add approvals there.

Reply:
ok

Reply:
https://github.com/Seagate/cortx-utils/pull/262
Raised.
Pls check and if ok then add approvals there